tests/test_class_attributes.rs

```diff
@@ -66,14 +66,43 @@ fn class_attributes() {
     });
 }
 
-// Ignored because heap types are not immutable:
-// https://github.com/python/cpython/blob/master/Objects/typeobject.c#L3399-L3409
 #[test]
-#[ignore]
+#[cfg(Py_3_14)]
 fn class_attributes_are_immutable() {
```

Comment:
Should we maybe add the opposite test, that classes without `immutable_type` _are_ immutable?

Reply:
Good idea to test that as well. Will add.